05-html-css/css-framework.md

```diff
@@ -18,185 +16,144 @@ The usual components are:
+We will take the expedient of using the Gem, and placing it into our [`.rails-template.rb`](https://github.com/Ada-Developers-Academy/textbook-curriculum/blob/master/09-intermediate-rails/reference/.rails-template.rb) file.
 
-In our [`rails-template.rb`](https://github.com/Ada-Developers-Academy/textbook-curriculum/blob/master/09-intermediate-rails/reference/.rails-template.rb) file we added code to automatically install the Foundation 6 framework.
+In our [`rails-template.rb`](https://github.com/Ada-Developers-Academy/textbook-curriculum/blob/master/09-intermediate-rails/reference/.rails-template.rb) file we added code to automatically install the Bootstrap 4 framework.
 
 The template adds the following content:
 
 ```ruby
 ...
 
-gem 'foundation-rails'
+gem 'bootstrap'
 
 ...
 
-# Run rails generate foundation:install
-  generate "foundation:install", "--force"
+# Add jquery to application.js to work with foundation-rails
+inject_into_file 'app/assets/javascripts/application.js', after: '// about supported directives.' do
+  <<-'JAVASCRIPT'
 
-...
+//= require jquery3
+//= require popper
+//= require bootstrap-sprockets
+  JAVASCRIPT
+end
 
-# Add Foundation Javascript with Motion-ui this must be run after
-# rails generate foundation:install, because that command
-# adds foundation_and_overrides.scss
-inject_into_file 'app/assets/stylesheets/foundation_and_overrides.scss', after: '// @include motion-ui-animations;' do
-<<-'RUBY'
+...
 
-@import 'motion-ui/motion-ui';
-@include motion-ui-transitions;
-@include motion-ui-animations;
+append_to_file 'app/assets/stylesheets/application.css' do
+  <<-'SCSS'
 
-RUBY
+/* Custom bootstrap variables must be set or imported *before* bootstrap. */
+@import "bootstrap";
+/* Import scss content */
+@import "**/*";
+  SCSS
 end
 ```
 
 These foundation template changes will cause Rails to:
-1. Add the `foundation-rails` gem to the gemfile.  
-2. Run a generator command to work with the gem to generate a bunch of CSS and JavaScript files.  
-3. Lastly add some code to `foundation_and_overrides.scss` in the stylesheets folder to include some libraries for animated menus and transitions.
-
-Now when you create a new Rails project you will see the following stylesheets.
-
-![Foundation Gem Stylesheets](imgs/foundation_stylesheets.png)
-
-You can change foundation settings by editing the `_settings.scss` file and turn off and on Foundation features by changing the `foundation_and_overrides.scss` file.  You can also override foundation styles, by defining css rules of your own in the `application.css` file.  
+1. Add the `bootstrap` gems to the gemfile.
+2. Tell Rails to import some JavaScript and CSS files for Bootstrap.
 
-## Using Foundation Typography
+## Using Bootstrap Typography
 
 You should first notice that foundation changes the default styling of all the header and paragraph text styling
 
-### Before Foundation:
+### Before Bootstrap:
 ![before-foundation-typography](imgs/before-foundation.png)
```

Comment:
Does this filename need to change?

Reply:
Also, I don't think this image is correct - pre-framework should be a serif font:

Here is an image with the right fonts.

<img width="340" alt="pre-framework-fonts" src="https://user-images.githubusercontent.com/21064499/46624831-bacbb000-cae6-11e8-912d-6c711921d511.png">

Reply:
I think if you'd like to not make these files need to be updated, if you followed your pattern of making two Codepens, that may be easier?